src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue

```diff
@@ -35,17 +34,17 @@ export default class ConsentContinuationOut extends Vue {
   }
 
   get orderDetails (): string {
-    return this.filing.data?.consentContinuationOut?.orderDetails
+    return this.filing.data?.order?.orderDetails?.replace('\n', '<br/>')
   }
 
   /** The court order file number. */
   get fileNumber (): string {
     return this.filing.data?.order?.fileNumber
   }
 
-  /** The court order plan of arrangement. */
-  get planOfArrangement (): string {
-    return this.filing.data?.order?.effectOfOrder ? 'Pursuant to a Plan of Arrangement' : null
+  /** Whether the court order has an effect of order. */
+  get hasEffectOfOrder (): boolean {
+    return Boolean(this.filing.data?.order?.effectOfOrder)
```

Comment:
Can we do `!!` instead of using `Boolean()` here to convert to boolean?

Reply:
Because of all the conditional chaining, I prefer this syntax.

I definitely like the `!!` syntax when you're looking at a single variable.

My rule: if it's simple, use `!!`, other use `Boolean(<not simple stuff here>)`.

Reply:
Got it. Makes a lot of sense. Thanks Sev!